artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java

```diff
@@ -176,6 +178,12 @@
  */
 public abstract class ActiveMQTestBase extends Assert {
 
+   public MBeanServer getMBeanServer() {
+      MBeanServer mBeanServer = MBeanServerFactory.createMBeanServer();
+      runAfter(() -> MBeanServerFactory.releaseMBeanServer(mBeanServer));
+      return mBeanServer;
+   }
```

Comment:
Renaming this something like createMBeanServer() would seem appropriate now, given its no longer doing the simple get it started out as in the previously reviewed version, and this method is only called in place of prior MBeanServerFactory.createMBeanServer() uses.

Reply:
It's getting you one from somewhere, right? :) 


but I can rename it .. sure